src/github/jsonRequest.js

```diff
@@ -18,5 +18,10 @@ export default function jsonRequest(access_token, token_type, method = "get", ur
   };
   if (data) option.body = JSON.stringify(data);
   return fetch(url + "?" + paramString, option)
-    .then(res => no_parse ? res : res.json());
+      .then(response => {
+          if (!response.ok) {
+              throw Error(response.statusText);
+          }
+          return response
+      }).then(res => no_parse ? res : res.json())
```

Comment:
I think a proper way to handle this would be to use `promise.reject` but I'm not quite sure.

Also besides, `response.ok` is false whenever (if I recall correctly) the status code is anything but `20x`. So I would probably leave this as it was for now and come back for more sophisticated handling later.